packages/kg-parser-plugins/lib/parser-plugins.js

```diff
@@ -510,6 +510,38 @@ export function createParserPlugins(_options = {}) {
         nodeFinished();
     }
 
+    // Nested paragraphs in blockquote are currently treated as separate blockquotes,
+    // see [here](https://github.com/bustle/mobiledoc-kit/issues/715). When running migrations,
+    // this is not the desired behaviour and will cause the content to lose the previous semantic.
+    function blockquoteWithChildren(node) {
+        if (node.nodeType !== 1 || node.tagName !== 'BLOCKQUOTE' || node.children.length < 1) {
+            return;
+        }
+
+        const html = [];
+        const children = Array.from(node.children);
+
+        children.forEach((child) => {
+            let nextSibling = child.nextSibling;
+            let previousSibling = child.previousSibling;
+
+            // Only add a soft-break for two sequential paragraphs.
+            // Use the innerHTML only in that case, so Mobiledoc's default behaviour
+            // of creating separate blockquotes doesn't apply.
+            if (child.tagName === 'P' && (nextSibling && nextSibling.tagName === 'P')) {
+                html.push(`${child.innerHTML}<br>`);
```

Comment:
Would `<br><br>` be more appropriate here to better match the intended display of separate paragraphs?

Reply:
Yup, makes sense! Just updated. Pasting html like this:

```html
<blockquote><p>My first quote line</p><p>My second quote line</p><p>My third quote line</p><p>My fourth quote line</p></blockquote>
```

results in this now:
![image](https://user-images.githubusercontent.com/8037602/97825517-c15b9e80-1d23-11eb-8f45-832f0e20414d.png)